src/pages/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import styled from "styled-components";
+
+import Button from "../../components/ui/Button";
+import GlobalStyle from "../../components/ui/GlobalStyle";
+
+const WelcomePage: React.FC = () => {
+  const [isPinned, setIsPinned] = React.useState<boolean>(false);
+  const [error, setError] = React.useState<string | null>(null);
+
+  const handleCheckIfPinned = React.useCallback((shouldError = true) => {
+    console.log("kartrak - check if pinned");
+
+    const checkIfPinned = async (): Promise<boolean> => {
+      const userSettings = await chrome.action.getUserSettings();
+
+      console.log("kartrak - userSettings", userSettings);
+
+      return userSettings.isOnToolbar;
+    };
+
+    checkIfPinned().then((isPinned) => {
+      if (isPinned) {
+        chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+          const activeTab = tabs[0];
+
+          if (activeTab) {
+            setIsPinned(true);
+          }
+        });
+      } else if (shouldError) {
+        setError("L'extension n'est pas épinglée à la barre d'outils.");
+      }
+    });
+  }, []);
+
+  const handleCloseTab = React.useCallback(() => {
+    console.log("kartrak - close tab");
+
+    chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+      const activeTab = tabs[0];
+
+      if (activeTab) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        chrome.tabs.remove(activeTab.id!);
+      }
+    });
+  }, []);
+
+  React.useEffect(() => {
+    handleCheckIfPinned(false);
+  }, []);
+
+  console.log("kartrak - isPinned", isPinned);
+
+  return (
+    <Container>
+      <Row>
+        <GlobalStyle />
+
+        {isPinned ? (
+          <>
+            <Title>{"Vous êtes prêt à y aller !"}</Title>
+
+            <TextContent>
+              {
+                "Vous pouvez désormais cliquer sur l’extension quand vous le souhaitez pour analyser des sites internets ou voir vos statistiques d’activités."
+              }
+            </TextContent>
+
+            <Image
+              src={require("../../assets/img/welcome/opened-extension.png")}
+              alt={"Extension ouverte"}
+            />
+
+            <StyledButton onClick={handleCloseTab}>
+              {"J’ai compris, merci"}
+            </StyledButton>
+          </>
+        ) : (
+          <>
+            <Title>
+              {"Kartrak est désormais installé sur votre navigateur"}
+            </Title>
+
+            <TextContent>
+              {
+                "Epinglez dès maintenant l’extension à la barre d’outils de votre navigateur. Vous serez amenés à ouvrir souvent l’extension, alors ça sera plus pratique !"
+              }
+            </TextContent>
+
+            <Image
+              src={require("../../assets/img/welcome/extension-pinned.png")}
+              alt={"Extension épinglée"}
```

Comment:
add width and height attributes to avoid blinking during first paint

Reply:
i don't understand

Reply:
an image should have a width and height attribute, what you. don't understand ?